guide-app/src/components/shared/ImageCard.js

```diff
@@ -0,0 +1,132 @@
+import React from "react";
+import {
+  StyleSheet,
+  View,
+  TouchableOpacity,
+  Text,
+  Image,
+  ImageSourcePropType
+} from "react-native";
+import LinearGradient from "react-native-linear-gradient";
+import { Colors, TextStyles } from "@assets/styles";
+import ImageView from "@shared-components/ImageView";
+
+type Props = {
+  size?: "expanded" | "compact",
+  image: string,
+  title: string,
+  subTitle?: string,
+  icons?: ImageSourcePropType[],
+  onPress: () => void
+};
+
+const ImageCard = ({
+  image,
+  onPress,
+  title = null,
+  subTitle = null,
+  size = "compact",
+  icons = []
+}: Props) => {
+  const height = size === "compact" ? 253 : 360;
+
+  return (
+    <View style={styles.container}>
+      <TouchableOpacity style={styles.buttonContainer} onPress={onPress}>
+        <ImageView style={[styles.image, { height }]} source={image} />
+        <LinearGradient
+          colors={["#00000000", "#000000bb"]}
+          style={styles.gradientContainer}
+        >
+          <View style={styles.contentContainer}>
+            <View style={styles.textContainer}>
+              <Text style={styles.titleLabel}>{title}</Text>
+
+              {subTitle !== null && (
+                <Text style={[styles.text, styles.subTitleLabel]}>
+                  {subTitle}
+                </Text>
+              )}
+            </View>
+
+            <View style={styles.iconContainer}>
+              {icons.map((icon, key) => (
+                <Image key={key} source={icon} style={styles.icon} />
+              ))}
+            </View>
+          </View>
+        </LinearGradient>
+      </TouchableOpacity>
+    </View>
+  );
+};
+
+const styles = StyleSheet.create({
+  container: {
+    borderRadius: 10,
+    shadowColor: "#000",
+    shadowOffset: {
+      width: 0,
+      height: 2
+    },
+    shadowOpacity: 0.25,
+    shadowRadius: 3.84,
```

Comment:
Getting these warnings: 

> View #2697 of type RCTView has a shadow set but cannot calculate shadow efficiently. Consider setting a background color to fix this, or apply the shadow to a more specific component.

Looks like it's this component but not sure what background color to apply

Reply:
Added `Colors.white` for now